### PR TITLE
Convert json in text to json in jsonb

### PR DESCRIPTION
**Pull request by hmpf**

### CODE REVIEW

File: src/easydmp/eestore/migrations/0002_convert_to_jsonb.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+def convert_to_jsonblob(apps, schema_editor):
+    EEStoreCache = apps.get_model('eestore', 'EEStoreCache')
+    for ec in EEStoreCache.objects.all():
+        if isinstance(ec.data, str):
+            ec.data = json.loads(ec.data)
+            ec.save()
+
+
+def convert_to_text(apps, schema_editor):
+    "Will be converted back by previous migration in operations list"
```
Comment (vbhavdal) [inline src/easydmp/eestore/migrations/0002_convert_to_jsonb.py:19]: Hm is that legal. :)

Comment (hmpf) [reply src/easydmp/eestore/migrations/0002_convert_to_jsonb.py:19]: If you migrate back to the migration before this one (0001_something) the field will be text in the database again. Tested :9 Only works when the two field types are comnpatible. Good thing too, as the "correct" way is very slow on the eventlog!

**Comment by hmpf** [comment]
I am shamelessly abusing this PR to figure out why we're testing on sqlite again btw.

**Comment by hmpf** [comment]
Closed in favor of PR #158.

**Pull request closed without merging**